src/intl/en/page-layer-2.json

```
@@ -40,19 +40,19 @@
-  "layer-2-rollups-2": "Rollups bundle (or ’roll up’) hundreds of transactions into a single transaction on layer 1. This distributes the L1 transaction fees across everyone in the rollup, making it cheaper for each user. Rollup transactions get executed outside of layer 1 but the transaction data gets posted to layer 1. By posting transaction data onto layer 1, rollups inherit the security of Ethereum. There are two different approaches to rollups: optimistic and zero-knowledge - they differ primarily on how this transaction data is posted to L1.",
+  "layer-2-rollups-1": "Rollups bundle (or ’roll up’) hundreds of transactions into a single transaction on layer 1. This distributes the L1 transaction fees across everyone in the rollup, making it cheaper for each user.",
+  "layer-2-rollups-2": "Rollup transactions get executed off layer 1 but the transaction data gets posted to layer 1. By posting transaction data onto layer 1, rollups inherit the security of Ethereum. This is because once the data is posted to layer 1, reverting a rollup transaction requires reverting Ethereum. There are two different approaches to rollups: optimistic and zero-knowledge - they differ primarily on how this transaction data is posted to L1.",
   "layer-2-optimistic-rollups-title": "Optimistic rollups",
   "layer-2-optimistic-rollups-description": "Optimistic rollups are 'optimistic' in the sense that transactions are assumed to be valid, but can be challenged if necessary. If an invalid transaction is suspected, a fault proof is run to see if this has taken place.",
   "layer-2-optimistic-rollups-childSentance": "More on optimistic rollups",
   "layer-2-zk-rollups-title": "Zero-knowledge rollups",
   "layer-2-zk-rollups-description": "Zero-knowledge rollups use validity proofs where transactions are computed off-chain, and then compressed data is supplied to Ethereum Mainnet as a proof of their validity.",
   "layer-2-zk-rollups-childSentance": "More on ZK-rollups",
   "layer-2-dyor-title": "Do your own research: risks of layer 2",
-  "layer-2-dyor-1": "Since layer 2 chains inherit security from Ethereum, in an ideal world, they are as safe as L1 Ethereum. However, many of the <strong>projects are still young and somewhat experimental</strong>. After years of research and development, many of the L2 technologies that will scale Ethereum launched in 2021. Many projects still have additional trust assumptions as they work to decentralize their networks. Always do your own research to decide if you're comfortable with any risks involved.",
+  "layer-2-dyor-1": "Many layer 2 projects are relatively young. Many projects still have additional trust assumptions as they work to decentralize their networks. Always do your own research to decide if you're comfortable with any risks involved.",
```

```suggestion
  "layer-2-dyor-1": "Many layer 2 projects are relatively young and still have additional trust assumptions as they work to decentralize their networks. Always do your own research to decide if you're comfortable with any risks involved.",
```
2x "many" used in a row